cron/eventStats.py

```diff
@@ -79,12 +80,21 @@ def main():
     logger.debug('starting')
     logger.debug(options)
     es = ElasticsearchClient((list('{0}'.format(s) for s in options.esservers)))
+    index = options.index
     stats = esSearch(es)
     logger.debug(json.dumps(stats))
+    sleepcycles = 0
     try:
-        # post to elastic search servers directly without going through
-        # message queues in case there is an availability issue
-        es.save_event(body=json.dumps(stats), doc_type='mozdefstats')
+        while es.index_exists(index) == False:
```

Comment:
The preferred way of doing this type of logic is to do:
```
while not es.index_exists(index):
   ...
```

Reply:
:) thanks! I'll correct.